src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseManagementForCaseWorkerService.java

```diff
@@ -100,22 +100,41 @@ private void respondentDefaults(CaseData caseData) {
             var respondentSumType = caseData.getRespondentCollection().get(0).getValue();
             caseData.setRespondent(nullCheck(respondentSumType.getRespondentName()));
             for (RespondentSumTypeItem respondentSumTypeItem : caseData.getRespondentCollection()) {
-                if (respondentSumTypeItem.getValue().getResponseReceived() == null) {
-                    respondentSumTypeItem.getValue().setResponseReceived(NO);
-                }
-                if (respondentSumTypeItem.getValue().getResponseReceived().equals(NO)
-                        && respondentSumTypeItem.getValue().getResponseRespondentAddress() != null) {
-                    resetResponseRespondentAddress(respondentSumTypeItem);
-                }
-                if (Strings.isNullOrEmpty(respondentSumTypeItem.getValue().getResponseContinue())) {
-                    respondentSumTypeItem.getValue().setResponseContinue(YES);
-                }
+                checkExtensionRequired(respondentSumTypeItem);
+                checkResponseReceived(respondentSumTypeItem);
+                checkResponseAddress(respondentSumTypeItem);
+                checkResponseContinue(respondentSumTypeItem);
             }
         } else {
             caseData.setRespondent(MISSING_RESPONDENT);
         }
     }
 
+    private void checkResponseAddress(RespondentSumTypeItem respondentSumTypeItem) {
+        if (respondentSumTypeItem.getValue().getResponseReceived().equals(NO)
```

Comment:
Can ResponseReceived be null? May be do this: (NO).equals(respondentSumTypeItem.getValue().getResponseReceived())

Reply:
ah this is called after checkResponseReceived() so it will not be null.